.travis.yml

```diff
@@ -20,7 +29,7 @@ branches:
 
 before_install:
   - (sudo apt-get update || brew update)
-  - (sudo apt-get install -y fontconfig libfribidi-dev yasm || (brew uninstall libtool; brew install fontconfig freetype fribidi libtool yasm)) && ./autogen.sh && ./configure
+  - (sudo apt-get install -y libfontconfig1-dev libfreetype6-dev libfribidi-dev libharfbuzz-dev yasm || brew install freetype fribidi harfbuzz yasm --without-gobject-introspection --without-icu4c) && ./autogen.sh && ./configure
```

Comment:
Running the whole `brew` command with the `--without-` options produces some benign warnings about formulæ other than `harfbuzz` not recognizing them. If this is undesired, I can split the command into two.